infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -413,7 +419,7 @@ sites:
       - herningbib.dk
     autogenerateRoutes: true
     plan: webmaster
-    <<: *webmasters-on-weekly-release-cycle
+    <<: *webmasters-staying-on-9.0
```

Comment:
We need to apply this (or a similar) anchor to Odense as well.

Due to local modifications on their production site [deployments have been failing lately](https://ui.lagoon.dplplat01.dpl.reload.dk/projects/odense/odense-main/deployments). This happens in the postdeploy step where the site state is migrated and configuration is imported. It is part luck that we have not had any code depend on updated state and thus causing failures.

With this is mind we need to keep Odense at their current version until the situation is resolved.

Reply:
ah... Rebekka didn't mention this, so good you knew